internal/coremsgs/en_struct_descriptions.go

```diff
@@ -376,55 +376,61 @@ var (
-	NetworkActionType           = ffm("NetworkAction.type", "The action to be performed")
-
-	// NodeStatus field descriptions
-	NodeNamespace  = ffm("NodeStatus.namespace", "The namespace that this status applies to")
-	NodeStatusNode = ffm("NodeStatus.node", "Details of the local node")
-	NodeStatusOrg  = ffm("NodeStatus.org", "Details of the organization identity that operates this node")
-	NodePlugins    = ffm("NodeStatus.plugins", "Information about plugins configured on this node")
-
-	// NodeStatusNode field descriptions
-	NodeStatusNodeName       = ffm("NodeStatusNode.name", "The name of this node, as specified in the local configuration")
-	NodeStatusNodeRegistered = ffm("NodeStatusNode.registered", "Whether the node has been successfully registered")
-	NodeStatusNodeID         = ffm("NodeStatusNode.id", "The UUID of the node, if registered")
-
-	// NodeStatusOrg field descriptions
-	NodeStatusOrgName       = ffm("NodeStatusOrg.name", "The name of the node operator organization, as specified in the local configuration")
-	NodeStatusOrgRegistered = ffm("NodeStatusOrg.registered", "Whether the organization has been successfully registered")
-	NodeStatusOrgDID        = ffm("NodeStatusOrg.did", "The DID of the organization identity, if registered")
-	NodeStatusOrgID         = ffm("NodeStatusOrg.id", "The UUID of the organization, if registered")
-	NodeStatusOrgVerifiers  = ffm("NodeStatusOrg.verifiers", "Array of verifiers (blockchain keys) owned by this identity")
-
-	// NodeStatusDefaults field descriptions
-	NodeStatusDefaultsNamespace = ffm("NodeStatusDefaults.namespace", "The default namespace on this node")
-
-	// NodeStatusPlugins field descriptions
-	NodeStatusPluginsBlockchain    = ffm("NodeStatusPlugins.blockchain", "The blockchain plugins on this node")
-	NodeStatusPluginsDatabase      = ffm("NodeStatusPlugins.database", "The database plugins on this node")
-	NodeStatusPluginsDataExchange  = ffm("NodeStatusPlugins.dataExchange", "The data exchange plugins on this node")
-	Events                         = ffm("NodeStatusPlugins.events", "The event plugins on this node")
-	NodeStatusPluginsIdentity      = ffm("NodeStatusPlugins.identity", "The identity plugins on this node")
-	NodeStatusPluginsSharedStorage = ffm("NodeStatusPlugins.sharedStorage", "The shared storage plugins on this node")
-	NodeStatusPluginsTokens        = ffm("NodeStatusPlugins.tokens", "The token plugins on this node")
-
-	// NodeStatusPlugin field descriptions
-	NodeStatusPluginName = ffm("NodeStatusPlugin.name", "The name of the plugin")
-	NodeStatusPluginType = ffm("NodeStatusPlugin.pluginType", "The type of the plugin")
+	NamespaceID                    = ffm("Namespace.id", "The UUID of the namespace. For locally established namespaces will be different on each node in the network. For broadcast namespaces, will be the same on every node")
```

Comment:
Do we need to change these descriptions to get rid of the `broadcast` terminology?